src/app/analyze/analyze.module.ts

```diff
@@ -1,16 +1,21 @@
-import { NgModule }         from '@angular/core';
-import { CommonModule }     from '@angular/common';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
 import { HttpModule, Http } from '@angular/http';
 
-import { AnalyzeComponent }     from './analyze.component';
+import { AnalyzeComponent } from './analyze.component';
 import { AnalyzeRoutingModule } from './analyze-routing.module';
 
 import { AnalyzeOverviewModule } from './analyze-overview/analyze-overview.module';
 
+import { StackDetailsModule } from './stack/stack-details/stack-details.module';
+
+
 @NgModule({
-  imports:      [ AnalyzeOverviewModule, CommonModule, AnalyzeRoutingModule, HttpModule ],
-  declarations: [ AnalyzeComponent ],
+  imports: [StackDetailsModule, AnalyzeOverviewModule,
```

Comment:
if you have to break a list into 2 lines, just take it all the way and make vertical. It will be easier to read if each Module is on it's own line.